### PR TITLE
[Doc] 3.1 Iceberg table sink - 1

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/data_source/catalog/iceberg_catalog.md [hunk 1]
```diff
@@ -793,12 +830,218 @@ You can use one of the following syntaxes to view the schema of an Iceberg table
...
+Similar to the internal tables of StarRocks, if you have the [DROP](../../administration/privilege_item.md#表权限-table) privilege on an Iceberg table, you can use the [DROP TABLE](../../sql-reference/sql-statements/data-definition/DROP%20TABLE.md) statement to drop that Iceberg table. This feature is supported from v3.1 onwards.
+
+> **NOTE**
+>
+> You can grant and revoke privileges by using [GRANT](../../sql-reference/sql-statements/account-management/GRANT.md) and [REVOKE](../../sql-reference/sql-statements/account-management/REVOKE.md).
+
+When you drop an Iceberg table, the table's file path and data on your HDFS cluster or cloud storage will not be dropped along with the table.
 
-Suppose you have an OLAP table named `olap_tbl`, you can transform and load data like below:
+When you forcibly drop an Iceberg table (namely, with the `FORCE` keyword specified in the table drop statement), the table's data on your HDFS cluster or cloud storage will be dropped along with the table, but the table's file path is retained.
```
Comment (evelynzhaojie) [inline docs/data_source/catalog/iceberg_catalog.md:1041]: table drop--- DROP TABLE

File: docs/data_source/catalog/iceberg_catalog.md [hunk 2]
```diff
@@ -793,12 +830,218 @@ You can use one of the following syntaxes to view the schema of an Iceberg table
...
+| column_name | The name of the destination column to which you want to load data. You can specify one or more columns. If specify multiple columns, separate them with commas (`,`). You can only specify columns that actually exist in the Iceberg table, and the destination columns that you specify must include the partition columns of the Iceberg table. The destination columns you specify are mapped one on one in sequence to the columns of the StarRocks table, regardless of what the destination column names are. If no destination columns are specified, the data is loaded into all columns of the Iceberg table. If a non-partition column of the StarRocks table cannot be mapped to any column of the Iceberg table, StarRocks writes the default value `NULL` to the Iceberg table column. If the INSERT statement contains a query statement whose returned column types differ from the data types of the destination columns, StarRocks performs an implicit conversion on the mismatched columns. If the conversion fails, a syntax parsing error will be returned. |
```
Comment (evelynzhaojie) [inline docs/data_source/catalog/iceberg_catalog.md:991]: If specify multiple columns --- If you specify multiple columns,

**Review by hellolilyliuyi — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by amber-create** [comment]
@mergify backport branch-3.1

**Pull request merged**